index.js

```diff
@@ -1,22 +1,15 @@
 'use strict';
-var fs = require('fs');
-var Promise = require('pinkie-promise');
+const fs = require('fs');
 
-module.exports = function (fp) {
-	var fn = typeof fs.access === 'function' ? fs.access : fs.stat;
-
-	return new Promise(function (resolve) {
-		fn(fp, function (err) {
-			resolve(!err);
-		});
+module.exports = fp => new Promise(resolve => {
+	fs.access(fp, err => {
+		resolve(!err);
```

Comment:
I'm conflicted about whether or not to put this inline. On one side it would look so much better, on another it doesn't return anything. Might be ok since we know it's a callback?